samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt

```diff
@@ -141,7 +142,8 @@ open class DefaultComponents(private val applicationContext: Context) {
                 LocationService.default()
             ),
             SearchMiddleware(applicationContext),
-            RecordingDevicesMiddleware(applicationContext)
+            RecordingDevicesMiddleware(applicationContext),
+            LastAccessMiddleware()
```

Comment:
Should we put up a PR for Fenix for this? It's easy to miss I guess as it won't complain at compile-time :).

Reply:
Yeah, I have it on my list. Will try to sync this up with an AC Nightly and update Fenix :)